lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +19,94 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-rows: auto auto;
+  grid-template-areas: 
+    "list-text actions"
+    "list-sub-text actions";
   margin: 0;
-  padding: $nx-spacing-sm $nx-spacing-xs;
+  padding: $nx-spacing-md;
 
   // we use first of type because we only want to modify the first list item,
   // not alerts, errors, or empty states
   &:first-of-type {
-    border-top: $nx-border;
+    border-top: 1px solid #d1d6f0;
   }
+}
 
-  &.nx-error {
-    border: $nx-border-error;
-    color: $nx-text-color-error;
-  }
+.nx-list__text {
+  @include semi-bold();
 
-  > .fa, > .svg-inline--fa {
-    margin-right: $nx-spacing-md;
-  }
+  grid-area: list-text;
 }
 
-.nx-list--bulleted {
-  .nx-list__item {
-    @include regular();
+.nx-list__subtext {
+  @include regular();
 
-    border: none;
-    list-style: disc outside;
-    margin: $nx-spacing-sm 10px $nx-spacing-sm 36px;
-    overflow: visible;
-    padding: 0;
-    white-space: normal;
-  }
+  display: block;
+  grid-area: list-sub-text;
+  // top margin is measured down from the top of the objects grid row
+  margin: $nx-spacing-md 0 0 0;
 
-  .nx-list__item {
-    list-style-type: disc;
+  &.nx-truncate-ellipsis {
+    white-space: nowrap;
+  }
+}
 
-    .nx-list__item {
-      list-style-type: circle;
+.nx-list__item--empty {
+  display: list-item;
+  text-align: center;
 
-      .nx-list__item {
-        list-style-type: square;
-      }
-    }
+  .nx-list__text {
+    @include regular();
   }
 }
 
-.nx-list__item--empty {
-  @include regular();
+.nx-list__item--error {
+  display: list-item;
 
-  border-top: $nx-border; // assume it's always the first item
-  border-bottom: $nx-border-row;
-  color: $nx-text-color-light;
-  text-align: center;
+  .nx-alert {
+    margin-top: 0;
+  }
 }
 
-.nx-list__item--disabled {
+.nx-list__item.disabled {
```

Comment:
Minor: is there a reason this isn't syntactically up in the main `.nx-list__item` section using `&.disabled`?

Reply:
> Minor: is there a reason this isn't syntactically up in the main `.nx-list__item` section using `&.disabled`?

Nope: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/f2e21198ea5496332e690d414f65c30e7f6c8bdb